### DISCUSSION TITLE
Crabada game update

### DISCUSSION THREAD

**Discussion by Slers** [coccoinomane/crabada.py#52] [Ideas]
Hi guys,

I guess you all saw the update pushed by the devs this morning. We will now need looting points that can only be acquired by mining (3 looting points per mine, 12 mining points max per team). My assumption is that this rule will balance the number of mines and loots so that the captcha will not be needed anymore to loot.

Hence, an ideal strategy would be to have a team that automatically mine once and then go loot three times (and repeat). What do you think?

Cheers,
Slers

**Comment by loayei** [comment]
Not possible. Points are tied to team.

**Reply by Slers** [reply]
You wouldn't change the team, but instead use the same team to mine/loot (for instance, with two bulks and one prime)

**Reply by loayei** [reply]
I understood you wrong I thought one will keep mining accumulating points for all others. But yeah your strategy is perfect but also You can just let them mine for 12 hours while u sleep (finally) and then loot for 12 hours. Wouldnt that be simpler thats taking into account captcha is still there

**Comment by coccoinomane** [comment]
Hi Slers!

Thank you for sharing the strategy, I think it makes sense as long as it is implemented on a per-team basis.

Is the Crabada team actually considering removing the captcha, if the update works as intended?

Cheers,
Guido

**Reply by Slers** [reply]
Hi Guido, thanks for your answer. I was hanging out on the Discord the other day and the team hinted something in this direction. They implemented the looting captcha as bots were detrimental to the game as mine supply was limited and regular people could hence not loot. But if people can now loot also without using a bot, the captcha has no use anymore. I guess that we will see in the coming days :)

**Reply by coccoinomane** [reply]
That's good to know, thanks for the update @Slers .

Let's stay tuned for developments: if they remove the captcha, either I or a contributor will surely implement a strategy taking advantage of the point system.

Cheers,
Cocco

**Comment by budibucus** [comment]
hi Cocco

Just curious on the crabada.py why should we set the node URI WEB3_NODE_URI="https://avalanche--mainnet--rpc.datahub.figment.io/apikey/<YOUR FIGMENT API KEY>/ext/bc/C/rpc"' on figment? and should my mm wallet connect to rpc from figment?

thanks

**Reply by coccoinomane** [reply]
Hello @budithegeekzwolf !

You can use any node provider, Figment is not special; I use it because its Avalanche nodes are fast and free to use.

About Metamask... not sure what you mean. You do not need a browser to use the bot :-)
